glean_parser/__main__.py

```diff
@@ -195,6 +196,23 @@ def coverage(coverage_file, metrics_files, format, output, allow_reserved):
     )
 
 
+@click.command()
+@click.argument("bug", type=str)
+@click.argument(
+    "metrics_files",
+    type=click.Path(exists=True, dir_okay=False, file_okay=True, readable=True),
+    nargs=-1,
+)
+def data_review_request(bug, metrics_files):
+    """
+    Generate a skeleton Data Review Request for all metrics in metrics_files
+    whose bug_numbers fields contain the provided bug string bounded by
+    non-word characters.
```

Comment:
Sounds very technical, maybe split that, call out an example like "e.g. the bug number" and mention "Bound by non-word characters" in another sentence?